deployment/aks-periscope.yaml

```
@@ -143,10 +143,10 @@ data:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: collectorflags-config
+  name: collectors-config
   namespace: aks-periscope
 data:
-  COLLECTOR_FLAGS_LIST: # <custom flag> connectedCluster OSM
+  COLLECTOR_LIST: # <custom flag> connectedCluster OSM
```

ref: https://github.com/Azure/aks-periscope/pull/48#discussion_r645234364
```suggestion
  COLLECTOR_LIST: managedCluster # <custom flag> connectedCluster OSM
```